Buildings/Controls/OBC/CDL/Continuous/Validation/MatrixGain.mo

```diff
@@ -0,0 +1,75 @@
+within Buildings.Controls.OBC.CDL.Continuous.Validation;
+model MatrixGain "Validation model for the MatrixGain block"
+
+  Buildings.Controls.OBC.CDL.Continuous.MatrixGain matGai(
+    final K=[1,0; 1,1; 1,2])
+    "Block that outputs the product of a matrix gain value with the input signal"
+    annotation (Placement(transformation(extent={{20,30},{40,50}})));
+
+  Buildings.Controls.OBC.CDL.Continuous.MatrixGain matGai1(
+    final K=[1,2; 3,4])
+    "Block that outputs the product of a matrix gain value with the input signal"
+    annotation (Placement(transformation(extent={{20,-50},{40,-30}})));
+
+protected
```

Comment:
Remove the `final` and `protected` so that users can run the example and change the values.